tokenserver/assignment/memorynode.py

```diff
@@ -19,14 +19,14 @@ def __init__(self, service_entry=None, **kw):
         self._service_entry = service_entry
         self._users = {}
         self._next_uid = 1
+        self.settings = kw or {}
 
     @property
     def service_entry(self):
         """Implement this as a property to have the context when looking for
         the value of the setting"""
         if self._service_entry is None:
-            settings = get_current_registry().settings
-            self._service_entry = settings.get('tokenserver.service_entry')
+            self._service_entry = self.settings.get('service_entry')
```

Comment:
I don't think this will ever do anything in practice - since the constructor takes `service_entry` as a positional argument, it should never appear in `**kw` and hence never appear in `self.settings`. I think we can just remove this property and use `self._service_entry` directly.

Reply:
true.